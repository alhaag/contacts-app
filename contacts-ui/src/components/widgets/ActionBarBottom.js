// third party
import React, { Component } from 'react'
import PropTypes from 'prop-types'
// styles
import style from '../../styles/widgets/action-bar-bottom.scss'


class ActionBarBottom extends Component {

    static propTypes = {
        children: PropTypes.node.isRequired,
        className: PropTypes.string,
    }

    render() {

        let { children } = this.props;

        return (
            <div className="bottom-action-bar">
                {children}
            </div>
        )
    }

}

export default ActionBarBottom
