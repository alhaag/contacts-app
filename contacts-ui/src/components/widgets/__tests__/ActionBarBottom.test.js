import React from 'react'
import renderer from 'react-test-renderer'
import ActionBarBottom from '../ActionBarBottom';

describe('AuthReducer', () => {

  test('Render AuthReducer default props', () => {
    const component = renderer.create(
      <ActionBarBottom>
        <buttom type="button">Test</buttom>
      </ActionBarBottom>,
    );
    let tree = component.toJSON();
    expect(tree).toMatchSnapshot();
  });

});