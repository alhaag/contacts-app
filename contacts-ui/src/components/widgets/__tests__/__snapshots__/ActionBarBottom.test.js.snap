// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`AuthReducer Render AuthReducer default props 1`] = `
<div
  className="bottom-action-bar"
>
  <buttom
    type="button"
  >
    Test
  </buttom>
</div>
`;
